libraries/dotnet/WorkbenchConnector/WorkbenchConfig.cs

```diff
@@ -9,14 +9,14 @@ public class WorkbenchConfig
     /// <summary>
     /// Semantic Workbench endpoint.
     /// </summary>
-    public string WorkbenchEndpoint { get; set; } = "http://127.0.0.1:3000";
+    public string WorkbenchEndpoint { get; set; } = "http://localhost:3000";
```

Comment:
do we need this change? we had a bunch of issues with this, and typically the IP works better